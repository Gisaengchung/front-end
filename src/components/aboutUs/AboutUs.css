.aboutContainer {
  background-color: rgb(25, 23, 23);
  box-shadow: 0px 0px 10px 10px rgba(0, 0, 0, 0.305);
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  grid-template-columns: 200px auto 200px;
  grid-template-rows: 200px auto 200px; 
  column-gap: 55px;
  text-align: center;
  align-items: center;
  align-content: center;
  justify-content: center;
  margin-top: 10%;
  margin-left: 4%;
  height: 70vh;
  width: 92vw;
  padding-right: 3%;
}

.personContainer {
  color: rgb(255, 255, 255);
}

.picContainer {
  padding: 20px;
  width: 17vw;
}

.ktImage {
  height: 250%;
  width: 125%;

}

.contactImage {
  height: 50px;
  width: 50px;
  padding: 7px;
}

span {
text-align: center;
padding-left: 4%;
padding-right: 4%;
}

.aboutNames {
  text-align: center;
}
