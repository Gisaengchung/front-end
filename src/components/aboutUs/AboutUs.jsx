/* eslint-disable max-len */
import React from 'react';
import styles from './AboutUs.css';
import kt from './aboutAssets/kt.jpg';
import kevin from './aboutAssets/kevin.png';
import rird from './aboutAssets/rird.png';
import sydney from './aboutAssets/sydney.png';
import github from './aboutAssets/github.png';
import linkedinlogo from './aboutAssets/linkedinlogo.png';


export default function AboutUs() {
  return (
    <div className={styles.aboutContainer}>
      <div className={styles.personContainer}>
        
        <div className={styles.picContainer}>
          <img className={styles.ktImage} src={rird} />
          <h2 className={styles.aboutNames}>Richard</h2>
          <span>Hi! My Name is Richard. In the future time is backwards!</span>
          <p/>
          <a href="https://github.com/Richard-Hillman">
            <img className={styles.contactImage} src={github} />
          </a>
          <a href="https://www.linkedin.com/in/richard-hillman/">
            <img className={styles.contactImage} src={linkedinlogo} />
          </a>
        </div>
      </div>

      <div className={styles.personContainer}>
        
        <div className={styles.picContainer}>
       
          <img className={styles.ktImage} src={kt} />
          <h2 className={styles.aboutNames}>Kt</h2>
          <span>Hi! My name is Katie! In my free time I create, direct, and edit fun films! </span>
          <p/>
          <a href="https://github.com/katiejonesyo">
            <img className={styles.contactImage} src={github} />
          </a>
          <a href="https://www.linkedin.com/in/katiejonesyo/">
            <img className={styles.contactImage} src={linkedinlogo} />
          </a>
        </div>
      </div>
      <div className={styles.personContainer}>
        
        <div className={styles.picContainer}>
          <img className={styles.ktImage} src={kevin} />
          <h2 className={styles.aboutNames}>Kevin</h2>
          <span>Hi! My name is Ferrari!I am Ferrari! Vroom vroom cars go fast yea yea yea</span>
          <p/>
          <a href="https://github.com/kevinfiero">
            <img className={styles.contactImage} src={github} />
          </a>
          <a href="https://www.linkedin.com/in/kevinfiero/">
            <img className={styles.contactImage} src={linkedinlogo} />
          </a>
        </div>
      </div>
      <div className={styles.personContainer}>

        <div className={styles.picContainer}>
          <img className={styles.ktImage} src={sydney} />
          <h2 className={styles.aboutNames}>Sydney</h2>
          <span>Hi! My name is Sydney! I am not tall!  I DO NOT DRINK COFFEEEEEEEEEE</span> 
          <p/>
          <a href="https://github.com/scjam">
            <img className={styles.contactImage} src={github} />
          </a>
  
          <a href="https://www.linkedin.com/in/sydjames/">
            <img className={styles.contactImage} src={linkedinlogo} />
          </a>
        </div>
      </div>
    </div>

  );
}


