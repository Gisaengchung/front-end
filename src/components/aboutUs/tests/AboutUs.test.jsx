import React from 'react';
import { render, cleanup } from '@testing-library/react';
import AboutUs from '../AboutUs';

describe('AboutUs component', () => {
  afterEach(() => cleanup());
  it('renders AboutUs', () => {
    const { asFragment } = render(<AboutUs />);
    expect(asFragment()).toMatchSnapshot();
  });
});
