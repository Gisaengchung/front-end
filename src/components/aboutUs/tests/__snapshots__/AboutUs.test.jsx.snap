// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs component renders AboutUs 1`] = `
<DocumentFragment>
  <div
    class="aboutContainer"
  >
    <div
      class="personContainer"
    >
      <div
        class="picContainer"
      >
        <img
          class="ktImage"
          src="/path/image"
        />
        <h2
          class="aboutNames"
        >
          Richard
        </h2>
        <span>
          Hi! My Name is Richard. In the future time is backwards!
        </span>
        <p />
        <a
          href="https://github.com/Richard-Hillman"
        >
          <img
            class="contactImage"
            src="/path/image"
          />
        </a>
        <a
          href="https://www.linkedin.com/in/richard-hillman/"
        >
          <img
            class="contactImage"
            src="/path/image"
          />
        </a>
      </div>
    </div>
    <div
      class="personContainer"
    >
      <div
        class="picContainer"
      >
        <img
          class="ktImage"
          src="/path/image"
        />
        <h2
          class="aboutNames"
        >
          Kt
        </h2>
        <span>
          Hi! My name is kt!I am tall!!!!!! WOOOOO!!!
        </span>
        <p />
        <a
          href="https://github.com/katiejonesyo"
        >
          <img
            class="contactImage"
            src="/path/image"
          />
        </a>
        <a
          href="https://www.linkedin.com/in/katiejonesyo/"
        >
          <img
            class="contactImage"
            src="/path/image"
          />
        </a>
      </div>
    </div>
    <div
      class="personContainer"
    >
      <div
        class="picContainer"
      >
        <img
          class="ktImage"
          src="/path/image"
        />
        <h2
          class="aboutNames"
        >
          Kevin
        </h2>
        <span>
          Hi! My name is Ferrari!I am Ferrari! Vroom vroom cars go fast yea yea yea
        </span>
        <p />
        <a
          href="https://github.com/kevinfiero"
        >
          <img
            class="contactImage"
            src="/path/image"
          />
        </a>
        <a
          href="https://www.linkedin.com/in/kevinfiero/"
        >
          <img
            class="contactImage"
            src="/path/image"
          />
        </a>
      </div>
    </div>
    <div
      class="personContainer"
    >
      <div
        class="picContainer"
      >
        <img
          class="ktImage"
          src="/path/image"
        />
        <h2
          class="aboutNames"
        >
          Sydney
        </h2>
        <span>
          Hi! My name is Sydney! I am not tall!  I DO NOT DRINK COFFEEEEEEEEEE
        </span>
        <p />
        <a
          href="https://github.com/scjam"
        >
          <img
            class="contactImage"
            src="/path/image"
          />
        </a>
        <a
          href="https://www.linkedin.com/in/sydjames/"
        >
          <img
            class="contactImage"
            src="/path/image"
          />
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
